server/events/policy_check_output_generator.go

```diff
@@ -0,0 +1,113 @@
+package events
+
+import (
+	"fmt"
+
+	"github.com/runatlantis/atlantis/server/events/command"
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/lyft/feature"
+)
+
+const KeySeparator = "||"
+
+type PolicyCheckOutputStore struct {
+	Store map[string]*models.PolicyCheckSuccess
+}
+
+func NewPolicyCheckOutputStore() *PolicyCheckOutputStore {
+	return &PolicyCheckOutputStore{
+		Store: map[string]*models.PolicyCheckSuccess{},
+	}
+}
+
+func (p *PolicyCheckOutputStore) buildKey(projectName string, workspace string) string {
+	return fmt.Sprintf("%s%s%s", projectName, KeySeparator, workspace)
+}
+
+func (p *PolicyCheckOutputStore) GetOutputFor(projectName string, workspace string) *models.PolicyCheckSuccess {
```

Comment:
Seems unnecessarily verbose

```suggestion
func (p *PolicyCheckOutputStore) Get(projectName string, workspace string) *models.PolicyCheckSuccess {
```